Description: Adjusting project file to use system libraries
 Patches mudlet.pro so that it uses pkgconfig to find standard libraries
 Removed references to luajit as it is not available to Debian yet
Bug-Debian: http://bugs.debian.org/680134
Author: Craig Small <csmall@debian.org>
Last-Update: 2017-07-09
--- a/src/mudlet.pro
+++ b/src/mudlet.pro
@@ -219,27 +219,16 @@
     isEmpty( BINDIR ) BINDIR = $${PREFIX}/bin
 # Again according to FHS /usr/local/share/games is the corresponding place for locally built games documentation:
     isEmpty( DOCDIR ) DOCDIR = $${DATAROOTDIR}/doc/mudlet
-    freebsd {
-        LIBS += \
-# Some OS platforms have a hyphen (I think Cygwin does as well):
-            -llua-5.1\
-# FreeFSB appends the version number to hunspell:
-            -lhunspell-1.6
-# FreeFSB (at least) supports multiple Lua versions (and 5.1 is not the default anymore):
-        INCLUDEPATH += \
-            /usr/local/include/lua51
-    } else {
-        LIBS += \
-            -llua5.1 \
-            -lhunspell
-        INCLUDEPATH += /usr/include/lua5.1
-    }
-    LIBS += -lpcre \
-        -L/usr/local/lib/ \
-        -lyajl \
-        -lGLU \
-        -lzip \
-        -lz
+
+    CONFIG += link_pkgconfig
+    PKGCONFIG += libpcre \
+	lua5.1 \
+	hunspell \
+	yajl \
+	glu \
+	libzip \
+	zlib
+
     LUA_DEFAULT_DIR = $${DATADIR}/lua
 } else:win32 {
     LIBS += -L"C:\\mingw32\\bin" \
